modules/nixos.nix

```diff
@@ -165,5 +164,36 @@ in {
           concatStringsSep " " ruleString
       ) (filter (f: f.enable && f.source != null) (attrValues files));
     }) (filterAttrs (_: u: u.files != {}) config.homes);
+    systemd.services = mapAttrs' (name: {files, ...}: {
+      name = "hjem-monitor-" + name;
+      value.enable = true;
+      value.wants = ["systemd-tmpfiles-setup.service" "nix-daemon.socket"];
+      value.after = ["nix-daemon.socket"];
+      value.wantedBy = ["default.target"];
+      value.description = "Monitoring for Hjem files";
+      value.serviceConfig = {
+        Type = "exec";
+        ExecStart = let
+          serviceScript = pkgs.writeScript "hjem-monitor-script" ''
+            #! ${pkgs.runtimeShell} -e
```

Comment:
Any reason we're not using `systemd.user.services.<name>.script` here?

Reply:
systemd user services are not managed by NixOS during activation, Hjem would need to manage that if we switched to user here

Reply:
the primary use is for it to trigger during a nixos-rebuild (so that, in the case of a conflict, it shows during the rebuild process), however it should also be triggering on boot, I will check that now.

Reply:
Will have a look at ``systemd.services.<name>.script`` now